librz/core/cbin.c

```diff
@@ -803,16 +803,14 @@ RZ_API bool rz_core_bin_apply_maps(RzCore *core, RzBinFile *binfile, bool va) {
 
 	RzListIter *it;
 	RzBinMap *map;
```

Comment:
BTW since number of maps is known we can reserve memory for them in the skyline vector to avoid reallocations.